src/allmydata/util/fileutil.py

```diff
@@ -1,9 +1,19 @@
-from __future__ import print_function
-
 """
+Ported to Python3
```

Comment:
For ease of grepping can you make this "Ported to Python 3." (to line up with the text itamar's been using).